etc/bin/swarm

```diff
@@ -204,7 +204,7 @@ append_config () {
         if prompt "Write \"$LINE\" to \"$LOC\"?"; then
             if [[ "$POS" == top ]]; then
                 # ... at the top of the file
-                echo -e "${LINE}\n$(cat ${LOC})" > "$LOC"
+                echo -e "${LINE}\n$(cat "$LOC")" > temp && mv temp "$LOC"
```

Comment:
I get the quoting, but is the temp file necessary?

Reply:
Funny, shellcheck was complaining about SC2094 locally

> Make sure not to read and write the same file in the same pipeline

But apparently not on GH Actions?

Reply:
Ah OK, my local shellcheck must be outdated, didn't see that.  However, going by my logic above, SC2094 does not apply for this particular construct.